All project code/views/pages/search.ejs

```diff
@@ -0,0 +1,17 @@
+<%- include ('../partials/header') %>
+    <%- include ('../partials/menu') %>
+
+        <main>
+            <div class="container">
+                <!--  -->
+                <h1>Search</h1>
+                <form action="/search" method="POST">
+                    <label for="location" class="form-label">Search for a Location</label>
```

Comment:
We need a city search box, and a country name search box, labels, ids, and names should be city and country respectively.

Reply:
Got it, sounds good.

Reply:
Just made a change which adds this, as well as a premade function for you to hook your API into the search bar.